src/test/java/org/minions/devfund/marylin/SquarelotronTest.java

```diff
@@ -0,0 +1,557 @@
+package org.minions.devfund.marylin;
+
+import org.junit.Test;
+
+import java.util.Arrays;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * SquarelotronTest class.
+ */
+public class SquarelotronTest {
+    private static final int ZERO = 0;
+    private static final int ONE = 1;
+    private static final int TWO = 2;
+    private static final int THREE = 3;
+    private static final int FOUR = 4;
+    private static final int FIVE = 5;
+
+    //Initialize squarelotrons with different sizes
+    private Squarelotron initialSquarelotronTwoByTwo = new Squarelotron(TWO);
+    private Squarelotron initialSquarelotronThreeByThree = new Squarelotron(THREE);
+    private Squarelotron initialSquarelotronFourByFour = new Squarelotron(FOUR);
+    private Squarelotron initialSquarelotronFiveByFive = new Squarelotron(FIVE);
+
+    /**
+     * Method to print a grid.
+     *
+     * @param grid  grid to print.
+     * @param title title to show.
+     */
+    void printGrid(final int[][] grid, final String title) {
+        System.out.println("========" + title + "=========");
+        for (int i = 0; i < grid.length; i++) {
+            for (int j = 0; j < grid.length; j++) {
+                System.out.print(" " + grid[i][j] + " ");
+            }
+            System.out.println();
+        }
+    }
+
+    /**********************************************************
+     **********************upsideDownFlip**********************
+     *********************************************************/
+
+    /**
+     * {{1, 2}, {3, 4}}.
+     */
+    @Test
+    public void testUpsideDownFlipTwoByTwoGridFirstRing() {
+        final int[][] expectedGrid = {{3, 4}, {1, 2}};
+        printGrid(initialSquarelotronTwoByTwo.getSquarelotron(), "INITIAL");
+        printGrid(expectedGrid, "EXPECTED");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "EXPECTED" 36 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)